eoc_journal/eoc_journal.py

```diff
@@ -84,6 +90,13 @@ def student_view(self, context=None):
         """
         context = context.copy() if context else {}
 
+        user = self._get_current_user()
+        course_id = getattr(self.runtime, 'course_id', 'course_id')
+        course_id = unicode(_normalize_id(course_id))
```

Comment:
I think we should duplicate what `problem-builder` does when storing `Answer` models: https://github.com/open-craft/problem-builder/blob/27e2f4ba5afdefcb6a4e5fd793969e1e626a0abe/problem_builder/answer.py#L56-L58
Apparently it doesn't normalize the `course_id` when storing, so we shouldn't normalize it either in this case.